phpcs/QualityAssurance/Sniffs/InfoFiles/RequiredSniff.php

```diff
@@ -62,12 +62,19 @@ public function process(File $phpcsFile, $stackPtr)
             return ($phpcsFile->numTokens + 1);
         }
 
-        if (isset($info['type']) === true
-            && (isset($info['php']) === false
-            || empty($info['php']) === true)
-        ) {
-            $error = '"php" property is missing in the info file';
-            $phpcsFile->addError($error, $stackPtr, 'PHP');
+        if (isset($info['name']) === false) {
```

Comment:
What about "core_version_requirement", is optional but in the case of non-existence of composer.json this is required.

Same for description.